routes/v0/cases.js

```diff
@@ -20,7 +39,11 @@ casesRouter.post('/', async (req, res) => {
   res.json(createdCase);
 });
 
-casesRouter.put('/:id', async (req, res) => {
+/**
+ * We use asyncHandler(fn) here because expressJS expects the middleware to be a synchronous function,
+ * but editCasePermissions(args) is asynchronous
+ * */
+casesRouter.put('/:id', asyncHandler(editCasePermissions), async (req, res) => {
```

Comment:
It's not clear to me that using middleware is best here.  We may want to capture both this read and the write in a transaction.  I'm not suggesting it needs to be changed now.  Just a thought.

Reply:
At first glance, I don't see a reason why the permission check and the update code should be part of a single transaction. But, it's super trivial to make the change you suggest.